.github/workflows/merge.yml

```diff
@@ -147,25 +148,28 @@ jobs:
               accounts:
                 task_definition:
                   container_definitions:
-                    accounts: &accounts_def
+                    accounts:
                       image: "$ECR_TAG"
               accounts-celery:
                 task_definition:
                   container_definitions:
                     accounts:
-                      image: "$CELERY_WORKER_ECR_TAG"
+                      image: "$ECR_TAG"
           EOF
 
           # Use yq to merge the stump into the main config
           yq -i '. *= load("newimage.yaml")' config.stage.yaml
 
           source ./venv/bin/activate
           pulumi login
-          pulumi stack select thunderbird/accounts/stage
+          pulumi stack select thunderbird/stage
           pulumi up -y --diff \
             --target 'urn:pulumi:stage::accounts::tb:fargate:FargateClusterWithLogging$aws:ecs/taskDefinition:TaskDefinition::accounts-stage-fargate-accounts-taskdef' \
             --target 'urn:pulumi:stage::accounts::tb:fargate:FargateClusterWithLogging$aws:ecs/taskDefinition:TaskDefinition::accounts-stage-fargate-accounts-celery-taskdef' \
             --target-dependents
+        
+        # TODO: Write up something that can monitor the deployments. We don't want to run the e2e tests until we know
+        #       the deployment has completed. Can we get the deployment ID from the pulumi run?
```

Comment:
Is this happening in this ticket or another? If the latter can you create an issue for it?